examples/pxScene2d/src/pxContext.h

```diff
@@ -127,17 +166,28 @@ class pxContext {
                  bool downscaleSmooth = false,
                  pxConstantsMaskOperation::constants maskOp= pxConstantsMaskOperation::NORMAL);
 
-  void drawEffect(float x, float y, float w, float h, pxTextureRef t, shaderProgram *shader);
+  void drawEffect(float x, float y, float w, float h, pxTextureRef t, shaderProgram *shader, void *user = NULL);
+
+  struct filterXYR
+  {
+    float x; float y; float r;
+  };
+  
+  pxContextFramebufferRef applyBlurSettings(pxContextFramebufferRef src, shadowFx_t *shdw,
+                                       filterXYR *filters, size_t count);
 
 #ifdef PXSCENE_FONT_ATLAS
   // This is intended to draw numQuads from the same texture.
   // vertices and uvs for the quads are passed in as if the quads will be rendered
   // using GL_TRIANGLES in an optimal way.  quad oriented backends can skip vertices appropriately
   // 6 vertices (12 floats) and 6 uvs (12 floats) per quad
   void drawTexturedQuads(int numQuads, const void *verts, const void* uvs,
-                          pxTextureRef t, float* color);
+                         pxTextureRef t, float* color, void *user = NULL);
 #endif                          
 
+  void drawTextEffects(int numQuads, const void *verts, const void* uvs,
+                      pxTextureRef t, textFx_t *textFx);
+
```

Comment:
Let's remove drawTextEffects from pxContext.h since it's not a a publicly used method (it's only used internally by pxContextGL).  We can put it inside of pxContextGL.cpp to keep pxContext.h concise.

Reply:
Hidden - static to pxContextGL.cpp